ogc-rs/src/gx.rs

```diff
@@ -0,0 +1,271 @@
+use core::ffi::c_void;
+
+/// Helper function for `Gx::init`
+pub fn gp_fifo(fifo_size: usize) -> *mut c_void {
+    unsafe {
+        let gp_fifo = crate::mem_cached_to_uncached!(libc::memalign(32, fifo_size));
+        libc::memset(gp_fifo, 0, fifo_size);
+        gp_fifo
+    }
+}
+
+type Mtx34 = [[f32; 4]; 3];
```

Comment:
Any reason this is not imported from ``gu.rs``?

Reply:
Sloppy code. Added `use crate::gx::Mtx34`.